src/interfaces/coral_web/src/components/Shared/Tooltip/Tooltip.tsx

```diff
@@ -142,11 +142,11 @@ export const Tooltip: React.FC<TooltipProps> = ({
             {...getFloatingProps()}
           >
             {size === 'sm' ? (
-              <Text styleAs="p-sm" className="text-marble-300">
+              <Text as="span" styleAs="p-sm" className="text-marble-300">
```

Comment:
don't ask me why but this fixes the tooltip being oddly tall
```suggestion
              <Text as="span" styleAs="p-sm" className="flex text-marble-300">
```

|after|before|
|-|-|
|<img width="286" alt="image" src="https://github.com/cohere-ai/cohere-toolkit/assets/5898755/22891875-e200-47a0-bd32-ff048b860f9d">|<img width="277" alt="image" src="https://github.com/cohere-ai/cohere-toolkit/assets/5898755/850889c5-da1b-4f96-b48b-4e3c460a4e56">|